Add remote execution/REX for Satellite (#542)
redhat-documentation/supplementary-style-guide: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

*See also*: xref:remote-access[remote access]

[[remote-execution]]
==== image:images/yes.png[yes] remote execution (noun, adjective)

> *Description*: In Red{nbsp}Hat Satellite, _remote execution_ is a feature that allows users to run jobs on hosts from Capsule by using shell scripts or Ansible. For example, a user can enable remote execution for Capsule and then run a remote execution job to apply errata to hosts.

Reviewer: @dfitzmau This change looks good to me.

> *Description*: In Red{nbsp}Hat Satellite, _remote execution_ is a feature that allows users to run jobs on hosts from Capsule by using shell scripts or Ansible. For example, a user can enable remote execution for Capsule and then configure a remote execution job to apply errata to hosts.

Reviewer 2: ```suggestion
*Description*: In Red{nbsp}Hat Satellite, _remote execution_ is a feature that allows users to run jobs on hosts from Capsule by using shell scripts or Ansible. For example, a user can enable remote execution for Capsule and then run a remote execution job to apply errata to hosts.
```
Nitpick.

*See also*: xref:ostree[OSTree]

[[rex]]
==== image:images/yes.png[yes] REX (noun, adjective)

> *Description*: In Red{nbsp}Hat Satellite, _REX_ is an acronym for _remote execution_, a feature that allows users to run jobs on hosts from Capsule by using shell scripts or Ansible. For example, a user can enable remote execution (REX) for Capsule and then run a REX job to apply errata to hosts.

Reviewer: Also, if you are defining the REX acronym in the first sentence, you don't need to redefine it in the second. How about rewriting it like this? 

```suggestion
*Description*: In Red{nbsp}Hat Satellite, _REX_ is an acronym for _remote execution_. You can use REX to run jobs on hosts from Capsule by using shell scripts or Ansible. For example, a user can enable REX for Capsule and then run a REX job to apply errata to hosts.
```
